Python/Product/Analysis/Interpreter/Ast/AstScrapedPythonModule.cs

```diff
@@ -157,8 +157,14 @@ class AstScrapedPythonModule : IPythonModule
                     fact.Log(TraceLevel.Info, "Scrape", proc.FileName, proc.Arguments);
 
                     proc.Start();
-                    var exitCode = proc.Wait(60000);
-                    
+                    int? exitCode = null;
+
+                    try {
+                        exitCode = proc.Wait(60000);
+                    } catch (Exception ex) {
+                        fact.Log(TraceLevel.Error, $"ScrapeException {ex.Message}", proc.FileName, proc.Arguments);
```

Comment:
Put the exception message as its own field. It makes processing the log file easier.